src/styles/card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  max-width: 200px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  display: flex;
```

Comment:
It's not necessary to use 'display: flex;' and 'flex-direction: column;' on the '.card' class. By default, block-level elements like div (which is probably what you're using for your card) display their children in a column. You can remove these properties unless you have a specific reason to use them.